src/QMCWaveFunctions/LCAO/AOBasisBuilder.h

```diff
@@ -171,8 +171,9 @@ bool AOBasisBuilder<COT>::putH5(hdf_archive& hin)
     hin.read(Normalized, "normalized");
     hin.read(Morder, "expandYlm");
     hin.read(basisName, "name");
-    hin.read(basisType, "type");
-    hin.read(addsignforM, "expM");
+    // Ye to Anouar: do we need these inputs? They are missing in current h5
+    //hin.read(basisType, "type");
```

Comment:
Not really (for now). But it is just because there is specialization for GAMESS and Pyscf... I mirrored the XML. If future codes (outside of the ones from molden, have something else we can always revisit. So It can be removed for now.

Reply:
OK. I will remove them.